vendor/github.com/containers/image/v5/docker/reference/normalize.go

```diff
@@ -13,6 +13,7 @@ var (
 	defaultDomain       = "docker.io"
 	officialRepoName    = "library"
 	defaultTag          = "latest"
+	UnknownDigestSuffix = "@@unknown-digest@@"
```

Comment:
This is now a syntax private to the `c/image/docker` transport, so it should be public in that package, not in the more shared `c/image/docker/reference`.